src/store/authorization/actions.ts

```diff
@@ -32,6 +33,13 @@ export const actions: ActionTree<AuthorizationStateInterface, RootStateInterface
       payload.successCallback(response);
     } catch (err: any) {
       payload.errorCallback(err);
+      const errorMessage = err.response.data.error_message;
+      const errorStatus = err.response.status;
```

Comment:
Isn't it easier to pass err object to error store, and parse it inside action of error store?

Reply:
Agree, it's much more cleaner.